app/web/server.rb

```diff
@@ -46,28 +46,24 @@ def json_params
 
     post '/todo/:todo_id' do
       command = Commands::Todo::Add::Command.new(json_params)
```

Comment:
I wonder if we should extract a `build` method here that `new`'s and `validate`'s...